weld/easy_ll/mod.rs

```diff
@@ -109,6 +111,35 @@ pub fn load_library(libname: &str) -> Result<(), LlvmError> {
     }
 }
 
+/// Outputs the target machine assembly based on the given engine and module using llvm-sys api.
+fn output_target_machine_assembly(engine: LLVMExecutionEngineRef, module: LLVMModuleRef) -> String {
+    // We create a pointer to a MemoryBuffer (MemoryBufferRef), and will pass its address to be
+    // modified by EmitToMemoryBuffer.
+    let mut output_buf : self::llvm::prelude::LLVMMemoryBufferRef = ptr::null_mut();
+    let mut err = ptr::null_mut();
+    unsafe {
+        let cur_target = LLVMGetExecutionEngineTargetMachine(engine);
+        let file_type = LLVMCodeGenFileType::LLVMAssemblyFile;
+        let res = LLVMTargetMachineEmitToMemoryBuffer(cur_target, module, file_type, &mut err, &mut output_buf);
+        if res == 1 {
+            let x = CStr::from_ptr(err as *mut c_char);
+            panic!("Getting LLVM IR failed! {:?}", x);
```

Comment:
Don't panic here: change the return type to WeldResult<String> and return an Err()